composer.json

```diff
@@ -20,8 +20,8 @@
     ],
     "require": {
         "php": ">=5.6.4",
-        "laravel/framework": ">=5.4|~7.0",
-        "hmaus/drafter-php": "^6.1.1",
+        "laravel/framework": ">=5.4",
```

Comment:
if it's for us, should we just enforce v9 as a minimum?

Reply:
It's not necessary. it can work with any laravel version if it has supported symfony/process version. But I can set it to 9, no problem